cms/static/cms/js/admin.base.js

```diff
@@ -18,3 +18,9 @@ const CMS = {
 // in case some data is already attached to the global CMS
 // we must not override it
 window.CMS = CMS.$.extend(window.CMS || {}, CMS);
+
+// make sure that jQuery is available as $ and jQuery
+if (!window.jQuery) {
+    window.jQuery = CMS.$;
```

Comment:
Minor nitpick, could be just written as:
`window.jQuery = window.$ = CMS.$`

Reply:
Yes, it can.